<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="MN141">
        <property name="minimumSize">
         <size>
          <width>165</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>165</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/MN141.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MN111">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/MN111.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos1">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos2">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos3">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos4">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos5">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos6">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos7">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pos8">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MN523">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/MN523.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="MN401">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/MN401.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="pult2">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pult3">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pult1">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pult4">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pult5">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pult6">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pult7">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pult8">
        <property name="minimumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>244</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
